apps/taskman/service.py

```diff
@@ -34,6 +34,9 @@
     Impact.NOVALUE: JiraPriority.UNDEFINED,
 }
 
+# The length of summary in a Jira task is restricted to 255
+JIRA_SUMMARY_MAX_LENGTH = 255
```

Comment:
Is there any reason to put the constant here? Taskman module has `constants.py` file